lib/http.js

```diff
@@ -171,9 +172,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
           });
       }
       const {profileAgentId} = req.params;
-      const {profileAgent} = await profileAgents.get({id: profileAgentId});
-
-      if(profileAgent.account === account) {
+      const {profileAgent, secrets} = await profileAgents.get(
+        {id: profileAgentId, includeSecrets: true});
+      // a profileAgent is claimed if there is an account set or
+      // it has a token for an integration
+      if(profileAgent.account || secrets.token) {
```

Comment:
This is not correct. We should not return success when there's a token, we should throw a `NotAllowedError` indicating that the profile agent cannot be claimed.

Reply:
@mattcollier warning I added one new commit to this branch. @dlongley ok I added this check here: https://github.com/digitalbazaar/bedrock-profile-http/pull/32/commits/43fd16990d843ca4bb7e5f13152dccf6c6af173e

Not sure on the error message wording. I made it somewhat vague as I'm not sure if we want a potential attacker to know the reason of the rejection.